### PR TITLE
Added new events-service and improved setup_repo script

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -36,17 +36,17 @@ services:
       - NODE_ENV=development
     volumes:
       - ./workspace-zen/cp-zen-platform:/usr/src/app:Z
-      - ./workspace-zen/cp-zen-frontend:/usr/src/app/node_modules/cp-zen-frontend:Z
-      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
+      - ./workspace-zen/cp-zen-frontend:/usr/src/app/node_modules/cp-zen-frontend:ro
```
Comment (butlerx) [inline docker-compose.yml]: The Z is for fedora selinux having stricter policies.

File: docker-compose.yml [hunk 2]
```diff
@@ -139,7 +140,15 @@ services:
     depends_on:
       - db
     volumes:
-      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
       - ./workspace-zen/cp-organisations-service:/usr/src/app:Z
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:ro
+  events2:
```
Comment (butlerx) [inline docker-compose.yml]: How is this started?

Comment (DanielBrierton) [reply docker-compose.yml]: Derp, yes, thank you.

File: docker-compose.yml (hunk 1 above)
Comment (DanielBrierton) [reply docker-compose.yml]: Only needed if we want to write to the folder, which is why I've gone `ro` in the first place. We've had an issue with our `.git` folder disappearing from cp-zen-frontend and cp-translations. Unable to reliably reproduce, but this seems likely to be the culprit given it doesn't happen to the services.

Comment (butlerx) [reply docker-compose.yml]: Ah yes and write is needed as you've a seperate file for yarn now

Comment (butlerx) [reply docker-compose.yml]: Good assumption

File: docker-compose.yml [hunk 3]
```diff
@@ -78,11 +79,12 @@ services:
       - CD_EVENTS=events
       - CD_BADGES=badges
       - POSTGRES_HOST=db
+      - CDF_ADMINS=admin@coderdojo.org,daniel@coderdojo.org
```
Comment (butlerx) [inline docker-compose.yml]: I get admin but should Daniel be Commited

Comment (Wardormeur) [reply docker-compose.yml]: I'm not sure we can cast Daniel as Code

Comment (butlerx) [reply docker-compose.yml]: Where does zen end and Daniel begin?

Comment (Wardormeur) [reply docker-compose.yml]: Are you trying to write a drama or a love story ? :p

**Pull request merged**